q2studio/api/types.py

```diff
@@ -35,5 +35,5 @@ def is_subtype():
 
 @types.route('/importable', methods=['POST'])
 def get_importable_types():
-    ret = [repr(t) for t in PLUGIN_MANAGER.importable_types]
+    ret = [repr(t).strip("'") for t in PLUGIN_MANAGER.importable_types]
```

Comment:
We don't need to compute the `repr` anymore (that's why this bug cropped up in the first place) - `PLUGIN_MANAGER.importable_types` used to return a [`set()` of semantic types](https://github.com/qiime2/qiime2/blob/1ef06fd6c0c78b41777afc7688d45e69d22a693b/qiime2/sdk/plugin_manager.py#L128-L140) - now its a dict where the keys are the `str` of the semantic type - so I think you can get away with this:

```suggestion
    ret = list(PLUGIN_MANAGER.importable_types.keys())
```

Reply:
This is much better!  Thanks :)